### PR TITLE
protonmail-bridge: 1.5.0 -> 1.6.9

### PR DESCRIPTION
**Pull request by thyol**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 117814` at e89a35e1 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>protonmail-bridge</li>
  </ul>
</details>
<details>
  <summary>3 suggestions:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`buildPhase` should probably contain `runHook preBuild`.

Near pkgs/development/go-modules/generic/default.nix:145:5:
```
    |
145 |     buildPhase = args.buildPhase or ''
    |     ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/missing-phase-hooks.md">missing-phase-hooks</a>

`installPhase` should probably contain `runHook preInstall` and `runHook postInstall`.

Near pkgs/development/go-modules/generic/default.nix:220:5:
```
    |
220 |     installPhase = args.installPhase or ''
    |     ^
```
</li>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`gpl3` is a deprecated license, please check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.

Near pkgs/applications/networking/protonmail-bridge/default.nix:33:5:
```
   |
33 |     license = licenses.gpl3;
   |     ^
```
</li>
  </ul>
</details>

**Comment by kalbasit** [comment]
>     * warning: [unclear-gpl](https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md)
>       `gpl3` is a deprecated license, please check if project uses `gpl3Plus` or `gpl3Only` and change `meta.license` accordingly.
>       Near pkgs/applications/networking/protonmail-bridge/default.nix:33:5:
>       ```
>          |
>       33 |     license = licenses.gpl3;
>          |     ^
>       ```

Seems like they use **gpl3Plus**. @thyol mind fixing that in a separate commit (same PR just another commit).

Excerpt from https://github.com/ProtonMail/proton-bridge/blob/master/COPYING_NOTES.md

> ProtonMail Bridge is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.

**Review by kalbasit — APPROVED** [review]

**Comment by kalbasit** [comment]
@rmcgibbo how to re-invoke your bot for a PR?

**Comment by rmcgibbo** [comment]
Unfortunately I haven't implemented that feature yet.

File: pkgs/applications/networking/protonmail-bridge/default.nix [hunk 1]
```diff
@@ -2,34 +2,43 @@
...
   buildInputs = [ libsecret ];
 
   buildPhase = ''
+    runHook preBuild
+
+    patchShebangs ./utils/
     make BUILD_TIME= build-nogui
+
+    runHook postBuild
   '';
```
Comment (SuperSandro2000) [inline pkgs/applications/networking/protonmail-bridge/default.nix:27]:
```suggestion
  preBuild = ''
    patchShebangs ./utils/
  '';
  
  makeFlags = [
    "BUILD_TIME="
    "build-nogui"
  ];
```

Comment (kalbasit) [reply pkgs/applications/networking/protonmail-bridge/default.nix:27]: I don't think makeFlags work with buildGoModule. I'm curious to find out.

Comment (thyol) [reply pkgs/applications/networking/protonmail-bridge/default.nix:27]: It seems to me that `makeFlags` is ignored by `buildGoModule`.

**Review by lightdiscord — APPROVED** [review]

File: pkgs/applications/networking/protonmail-bridge/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/applications/networking/protonmail-bridge/default.nix:27]: Oh, then we need to add ``-j$NIX_BUILD_CORES``.

**Comment by OPNA2608** [comment]
[1.6.9 was released](https://github.com/ProtonMail/proton-bridge/releases/tag/br-1.6.9), could you bump the PR to that?

File: pkgs/applications/networking/protonmail-bridge/default.nix (hunk 1 above)
Comment (thyol) [reply pkgs/applications/networking/protonmail-bridge/default.nix]: Done, thank you for your feedback.

**Pull request merged**